docs/common-reasons-for-ads-not-showing.mdx

```diff
@@ -0,0 +1,63 @@
+# Common reasons for ads not showing
+
+Use the following guide to understand common reasons why apps show few or no ads.
+
+## Did you recently create your AdMob account?
+
+Wait up to 24 hours after you create your account.
+When you first sign up for AdMob, your account is reviewed before it’s approved.
+This typically takes up to 24 hours, but in rare cases can take up to 2 weeks.
+You’ll be notified via email if your account is not approved by AdMob.  
```

Comment:
Are you also notified if you are approved? Stated differently, are you notified either way? If you are notified either way, then it should read "You'll be notified of approval or rejection via email after review". If you are only notified on rejection then it should be something like "Check your account dashboard (if that exists? No idea, where ever you would check) to see if you are approved. If you are rejected you will be notified via email"

Reply:
Please note that I took most texts from Google Answers: https://support.google.com/admob/answer/9469204